cubequery/tasks/__init__.py

```diff
@@ -352,7 +369,22 @@ def check_float(s):
         return True
     except ValueError:
         return False
-    
 
+      
+def check_float_range(s, param):
+    if not param.valid:
+        return True
+
+    if not isinstance(param.valid, list):
+        return True
+
+    if len(param.valid) == 2:
+        pass
+    else:
+        pass
+
+    v = float(s)

```

Comment:
I don't understand the purpose of this function, specially the last part where whatever the len is the process 'pass' to assign the float to v variable

Reply:
Good spot, clearly some incomplete code.

It should be checking min max if there are two entries, and its one of the values if there are more than two